mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -2737,10 +2737,31 @@ def test_calculation_of_date_before_which_checks_need_attention(self):
             parsed_qs = parse_qs(api_call_made.split('?', 1)[1])
             self.assertEqual(parsed_qs['started_at__lt'], [form.need_attention_date.strftime('%Y-%m-%d %H:%M:%S')])
 
+    def test_credit_row_has_review_link(self):
+        """
+        Test that the view displays link to review a credit.
+        """
+        with responses.RequestsMock() as rsps:
+            self.login(rsps=rsps)
+            self.mock_need_attention_count(rsps, 0)
+            rsps.add(
+                rsps.GET,
+                api_url('/security/checks/'),
+                json={
+                    'count': 1,
+                    'results': [self.SAMPLE_CHECK],
+                },
+            )
+
+            response = self.client.get(reverse('security:check_list'), follow=True)
 
-class AcceptCheckViewTestCase(BaseCheckViewTestCase):
+            content = response.content.decode()
+            self.assertIn('Review <span class="visually-hidden">credit to John Doe</span>', content)
```

Comment:
tiny fyi: there's a slightly simpler `assertContains` which does the content decoding already by itself